src/NuGet.Clients/VsExtension/Commands/RestorePackagesCommand.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Design;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Shell;
+using NuGet.PackageManagement;
+using NuGet.PackageManagement.VisualStudio;
+using NuGetConsole;
+
+namespace NuGetVSExtension
+{
+    /// <summary>
+    /// Restore packages menu command handler.
+    /// </summary>
+    internal sealed class RestorePackagesCommand
```

Comment:
>RestorePackagesCommand [](start = 26, length = 22)

Can we mention this to the test team? This should be added to the manual test pass.